.changeset/filter-relationships-too.md

```diff
@@ -0,0 +1,5 @@
+---
+"@keystone-6/fields-document": patch
+---
+
+Fix the relationship fields to default to `initialFilter` and `initialSort` from the foreign list (if provided)
```

Comment:
This is primarily a patch for `initialSort`, as `initialFilter` is only added in https://github.com/keystonejs/keystone/pull/9629